### DISCUSSION TITLE
环境变量没有介绍如何使用

### DISCUSSION THREAD

**Discussion by xiegoudan** [1715173329/UnblockNeteaseMusic#464] [Q&A] [answered]
环境变量没有介绍如何使用，把env-example 复制成为 .env 后变并更改端口后无效，使用 node .\app.js 默认端口还是为8080
配置为windows服务也没有环境变量在哪儿编辑的入口

**Comment by 1715173329** [comment, marked as answer]
`env-example` 已经弃用，参数应该在 `nw.js` 中进行指定。

环境变量，你可以在系统设置中进行指定，或者参照 https://github.com/coreybutler/node-windows#environment-variables

**Comment by YuChaoM** [comment]
docker 运行怎么配置环境变量呢
